src/System.ComponentModel.Annotations/src/System/ComponentModel/DataAnnotations/DisplayFormatAttribute.cs

```diff
@@ -27,9 +32,35 @@ public DisplayFormatAttribute()
         public string DataFormatString { get; set; }
 
         /// <summary>
-        ///     Gets or sets the string to display when the value is null
+        ///     Gets or sets the string to display when the value is null, which may be a resource key string.
+        ///     <para>
+        ///         Consumers should use the <see cref="GetNullDisplayText" /> method to retrieve the UI display string.
+        ///     </para>
         /// </summary>
-        public string NullDisplayText { get; set; }
+        /// <remarks>
+        ///     The property contains either the literal, non-localized string or the resource key
+        ///     to be used in conjunction with <see cref="NullDisplayTextResourceType" /> to configure a localized
+        ///     name for display.
+        ///     <para>
+        ///         The <see cref="GetNullDisplayText" /> method will return either the literal, non-localized
+        ///         string or the localized string when <see cref="NullDisplayTextResourceType" /> has been specified.
+        ///     </para>
+        /// </remarks>
+        /// <value>
+        ///     The null dispay text is generally used as placeholder when the value is not specified.
+        ///     A <c>null</c> or empty string is legal, and consumers must allow for that.
+        /// </value>
+        public string NullDisplayText
+        {
+            get { return _nullDisplayText.Value; }
+            set
+            {
+                if (_nullDisplayText.Value != value)
```

Comment:
This and the `NullDisplayTextResourceType` setter match what `[Display]` properties do. But, I'm not sure why (@lajones ??). Duplicates checks in the `LocalizableString` properties.